jp.js

```diff
@@ -173,8 +173,8 @@ settings.interface.hitInd ヒットマーカーを表示
 settings.interface.hitColor ヒットマーカーの色
 settings.interface.dmg ダメージを表示
 settings.interface.dmgType Show Damage
-settings.interface.dmgType.default Rounded
-settings.interface.dmgType.float Float
+settings.interface.dmgType.default 切り上げ
+settings.interface.dmgType.float 
```

Comment:
this shouldn't be blank.

both `settings.interface.dmgType.default` and `settings.interface.dmgType.float` are referring to number rounding

eg. rounded up or down for `rounded` (99.8 would become 100 and be displayed), where as a `float` is the exact number with a decimal (99.8 would be displayed)